livy/README.md

```diff
@@ -0,0 +1,26 @@
+# Apache Livy Initialization Action
+
+This [initialization action](https://cloud.google.com/dataproc/init-actions) installs version
+`0.5.0` of [Apache Livy](https://livy.incubator.apache.org/) on a master node within a
+[Google Cloud Dataproc](https://cloud.google.com/dataproc) cluster.
+
+## Using this initialization action
+You can use this initialization action to create a new Dataproc cluster with Livy installed by:
+
+1. Uploading a copy of this initialization action (`livy.sh`) to
+  [Google Cloud Storage](https://cloud.google.com/storage).
+2. Using the `gcloud` command to create a new cluster with this initialization action.
+  The following command will create a new cluster named `<CLUSTER_NAME>` and specify the
+  initialization action stored in `<GCS_BUCKET>`
+
+    ```bash
+    gcloud dataproc clusters create <CLUSTER_NAME> \
+    --initialization-actions gs://<GCS_BUCKET>/livy.sh
+    ```
+Alternatively, you can start a regular dataproc cluster,
+[ssh to the master node](https://cloud.google.com/dataproc/submit-job) (see SSH into instance),
+clone this repository and run ./livy.sh (as sudo)
+
+3. Once the cluster has been created, Livy is configured to run on port `8998` on the master node
+in a Dataproc cluster.
+
```

Comment:
A hello world example (POSTing a pyspark job from /usr/lib/spark/examples?) and/or link to Livy user docs would be nice

Reply:
Added doc link